nexus/db-queries/src/db/datastore/mod.rs

```diff
@@ -189,6 +189,21 @@ impl<U, T> RunnableQuery<U> for T where
 {
 }
 
+/// Specifies whether the consumer wants to check whether they're allowed to
```

Comment:
Same -- this seems like it should be in #8925

Reply:
Moved into https://github.com/oxidecomputer/omicron/pull/8931